languages/go/config.json

```diff
@@ -141,7 +140,7 @@
         "slug": "nil",
         "uuid": "7b265cc8-8f88-4de1-b653-10a384e6c7e2",
         "concepts": ["nil"],
-        "prerequisites": ["basics", "maps", "pointers", "channels"]
+        "prerequisites": ["basics"]
```

Comment:
`nil` is an early exercise and cannot depend on `maps`, `pointers`, etc. as they depend on the `nil` exercise.